Resources/Prototypes/Procedural/salvage_rewards.yml

```diff
@@ -7,7 +7,6 @@
     CrateMaterialPlastic: 1.0
     CrateMaterialSteel: 1.0
     # things the station might want
-    CrateEngineeringAMEJar: 0.25
```

Comment:
comment this out instead of removing, with deltav comment
also ive never seen this i think its unused